polygon-digital-carbon/subgraph.template.yaml

```diff
@@ -1,13 +1,13 @@
-specVersion: 0.0.4
+specVersion: 0.0.8
 description: Polygon Carbon
 repository: https://github.com/KlimaDAO/klima-subgraph
 schema:
   file: ./schema.graphql
-features:
-  - grafting
-graft:
-  base: QmUTK4dB7CXVvN2XsX1temzZGcryQ59j8YKdQP5Z2AwzVZ
-  block: 58282296
+# features:
```

Comment:
could you throw in a comment about how this grafting block should/shouldn't be used?